src/plots/linePlot.ts

```diff
@@ -138,6 +139,29 @@ export module Plots {
       return this;
     }
 
+    /**
+     * Gets if croppedRendering is enabled
+     *
+     * When croppedRendering is enabled, lines that will not be visible in the viewport will not be drawn.
+     * Thus if only part of the data is in the viewport, then this option will boost of rendering.
+     * However if all the data will render, enabling will cause an insignificant overhead.
+     */
+    public croppedRendering(): boolean;
```

Comment:
I don't have a strong preference either, I just want to double check cuz I feel like that's not what people preferred in the RFC